pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -68,6 +72,18 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 		opts := new(clientconfig.ClientOpts)
 		opts.Cloud = ic.Config.Platform.OpenStack.Cloud
 		_, err = clientconfig.GetCloudFromYAML(opts)
+
+		// GetCloudFromYAML sequentially checks several places for the presence
+		// of clouds.yaml file. Unfortunately, it does not provide an interface
+		// to understand from which specific location this file was read. Thus, we
+		// have to look through the presence of files in the same order and determine
+		// the location by ourselves.
+		configPath := determineOpenStackConfigLocation()
```

Comment:
Feels weird to redo the job. And if the implementation of `GetCloudFromYAML` changes, we will be potentially logging the wrong location.

What if we vendor (aka copy-paste) Gophercloud's `GetCloudFromYAML` instead, and build the  `clientconfig.Cloud` ourselves? It looks like it only has exported properties.